index.html

```diff
@@ -1,117 +1,158 @@
+              <li class="list-item">Travelling</li>
+              <li class="list-item">Cooking</li>
+              <li class="list-item">Sustainability</li>
+            </ul>
+          </section>
+        </div>
+        <div class="wrapper">
+          <section id="life" class="section-box">
             <h3>Biography</h3>
-            <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
-        </section>
-        <section class="section-box">
-          <div class= "section-cv-flex">
-            <div class="cv-column">
-              <h3>2020</h3>
-            </div>
-            <div class="cv-column">
-              <h4>Web Development Bootcamp</h4>
-              <p>neuefische GmbH, Germany</p>
-            </div>
-            <div class="cv-column">
-              <h3>2020 - 2021</h3>
-            </div>
-            <div class="cv-column">
-              <h4>Personal Assistant</h4>
-              <p>Various employers, Germany</p>
-            </div>
-            <div class="cv-column">
-              <h3>2013 - 2019</h3>
-            </div>
-            <div class="cv-column">
-              <h4>Dive Center Manager</h4>
-              <p>TGI Helengeli, Maldives</p>
-            </div>
-            <div class="cv-column">
-              <h3>2007 - 2010</h3>
-            </div>
-            <div class="cv-column">
-              <h4>B.Sc. in Biology</h4>
-              <p>University of Cologne, Germany<p>
-            </div>
-          </div>
-        </section>
-      </div>
-    </main>
-    <footer>
-      <div class="footer">
+            <p>
+              Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+              nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+              erat, sed diam voluptua. At vero eos et accusam et justo duo
+              dolores et ea rebum. Stet clita kasd gubergren, no sea takimata
+              sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit
+              amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor
+              invidunt ut labore et dolore magna aliquyam erat, sed diam
+              voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
+              Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum
+              dolor sit amet.
+            </p>
+          </section>
+          <section class="section-box alignment">
+            <dl>
```

Comment:
Nice! Love the `<dl>`. Changed my code the same way!

Reply:
Yes, so much cleaner, hehe. Thank you!